extensions/wikia/FounderEmails/events/FounderEmailsEditEvent.class.php

```diff
@@ -34,94 +34,83 @@ public function __construct( Array $data = array() ) {
 		$this->setData( $data );
 	}
 
+	/**
+	 * @param array $events
+	 *
+	 * @return bool
+	 */
 	public function process( Array $events ) {
```

Comment:
Simplified this quite a bit and moved the functionality into smaller methods.  It might be useful to look at the full file in addition to the changeset to see how it all fits together.